CONTRIBUTING.md

```diff
@@ -8,11 +8,9 @@ Pull requests will be accepted based on the following:
 1. If the company is at least partially remote
 1. If you add the company in alphabetical order in the list
 1. If you submit the company with a website
-1. If you submit the company with the regions that are accepted for remote positions (US only? Worldwide? Specific timezones?) \*
+1. If you submit the company with the regions that are accepted for remote positions (for more details see the instructions in the [example company profile](/company-profiles/example.md))
```

Comment:
Let's make this link to `/company-profiles/example.md#region` so that it goes directly to the right section.